.github/workflows/build.yaml

```
@@ -46,7 +46,7 @@ jobs:
     needs: setup
     strategy:
       matrix:
-        runner: [windows-large, macos-12-xl]
+        runner: [windows-large, macos-14-xlarge]
         configuration: ${{ fromJSON(needs.setup.outputs.configurations) }}
         include:
         - runner: linux-large
```

```suggestion
        - runner: linux-large
          continue-on-error: true 
```
While we're at it, let's ensure failed Linux builds don't cancel others.